docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -300,310 +438,325 @@ In most cases, the same extended comparators are available for both lifting and
+*Filter the current working set to include only files (file:bytes nodes) with a PDB path containing the string 'tekide':*
 
-- Filter results to include only mutexes that start with the string “Net”:
+.. storm-pre:: [ (file:bytes=sha256:1a287331e2bfb4df9cfe2dab1b77c9b5522e923e52998a2b1934ed8a8e52f3a8 :mime='application/vnd.microsoft.portable-executable' :mime:pe:pdbpath='C:\Users\mr.tekide\Documents\Visual Studio 2013\Projects\njrat7stubsoures – Copy\njrat7stubsoures\obj\Debug\dvvm.pdb') ] +file:bytes:mime:pe:pdbpath ~= tekide
+::
+  
+  <query> +file:bytes:mime:pe:pdbpath ~= tekide
 
-.. storm-pre:: [it:dev:mutex=Network]  +it:dev:mutex ~= "^Net"
+.. storm-pre:: [ (file:bytes=sha256:1a287331e2bfb4df9cfe2dab1b77c9b5522e923e52998a2b1934ed8a8e52f3a8 :mime='application/vnd.microsoft.portable-executable' :mime:pe:pdbpath='C:\Users\mr.tekide\Documents\Visual Studio 2013\Projects\njrat7stubsoures – Copy\njrat7stubsoures\obj\Debug\dvvm.pdb') ] +:mime:pe:pdbpath ~= tekide
 ::
-    
-    <query> +it:dev:mutex ~= "^Net"
+  
+  <query> +:mime:pe:pdbpath ~= tekide
 
 
-**Usage Notes:**
+*Filter the current working set to exclude organizations (ou:org nodes) whose name contains a string that starts with "v", followed by 0 or more characters, followed by "x":*
+
+.. storm-pre:: [ (ou:org=4b0c2c5671874922ce001d69215d032f :name="the vertex project" :alias=vertex) (ou:org=ad8de4b5da0fccb2caadb0d425e35847 :name=vxunderground) ] -ou:org:name ~= '^v.*x'
+::
+  
+  <query> -ou:org:name ~= '^v.*x'
+
+.. storm-pre:: [ (ou:org=4b0c2c5671874922ce001d69215d032f :name="the vertex project" :alias=vertex) (ou:org=ad8de4b5da0fccb2caadb0d425e35847 :name=vxunderground) ] -:name ~= '^v.*x'
+::
+  
+  <query> -:name ~= '^v.*x'
 
-- Filtering using regular expressions is performed by matching the regex against the relevant property
-  of each node in the working set. Note that **prefix filtering** (see below) is supported for string
-  types and can be used as a more efficient alternative in some cases.
 
 .. _filter-prefix:
 
 Filter by Prefix (^=)
 +++++++++++++++++++++
 
-Synapse performs prefix indexing on strings (and string-derived types), which optimizes filtering nodes
-whose *<valu>* or *<pval>* starts with a given prefix. The extended comparator ``^=`` is used to filter
-nodes by prefix.
+Synapse performs prefix indexing on strings and string-derived types, which optimizes filtering nodes whose *<valu>*
+or *<pval>* starts with a given prefix (substring). The extended comparator ``^=`` is used to filter nodes by prefix.
+
+.. NOTE::
+  
+  Extended string types that support dotted notation (such as the :ref:`type-loc` or :ref:`type-syn-tag` types)
+  have custom behaviors with respect to lifting and filtering by prefix.
+  
+  :ref:`type-inet-fqdn` nodes are indexed in reverse string order so cannot be filtered using the prefix extended
+  operator. However, reverse indexing allows wildcard ( ``*`` ) matching of the beginning of any FQDN string.
```

Comment:
Filtering FQDNs by prefix does work, it is only lifting that doesn't allow it.

Reply:
Derp. At least this was noted in the type-specific docs!